client/my-sites/upgrades/checkout-thank-you/jetpack-thank-you-card.jsx

```diff
@@ -164,105 +225,69 @@ class JetpackThankYouCard extends Component {
 		getPluginFromStore();
 	}
 
-	renderPlugin( key = 0, plugin ) {
-		const classes = classNames( 'checkout-thank-you__jetpack-plugin', {
-			'is-placeholder': ! plugin
+	renderFeature( feature, key = 0 ) {
+		const classes = classNames( 'checkout-thank-you__jetpack-feature', {
+			'is-placeholder': ! feature
 		} );
-		return (
-			<div key={ key } className={ classes } >
-				<div className="checkout-thank-you__jetpack-plugin-status-icon">
-					<span>
-						{ plugin
-							? this.getStatusIcon( plugin )
-							: this.getStatusIcon( { status: 'placeholder' } )
-						}
-					</span>
-				</div>
-				<div className="checkout-thank-you__jetpack-plugin-status-text">
-					<span>
-						{ plugin
-							? this.getStatusText( plugin )
-							: this.getStatusText( {
-								slug: 'placeholder',
-								status: 'placeholder'
-							} )
-						}
-					</span>
-				</div>
-			</div>
-		);
-	}
 
-	renderFeaturePlaceholders() {
-		const placeholderCount = !! this.props.whitelist ? 1 : 3;
-		return range( placeholderCount ).map( i => {
-			return this.renderPlugin( i );
-		} );
-	}
+		const description = feature
+			? get( FEATURES_LIST, [ feature.slug, 'getTitle' ], () => false )()
+			: 'Activating your Jetpack plan';
```

Comment:
It's used as placeholder text, which will not be shown to the user. So, I don't believe it should be translated.